modules/layers/src/bitmap-layer/bitmap-layer.ts

```diff
@@ -141,11 +143,11 @@ export default class BitmapLayer<ExtraPropsT extends {} = {}> extends Layer<
     model?: Model;
     mesh?: any;
     coordinateConversion: number;
-    bounds: number[];
+    bounds: [number, number, number, number];
   };
 
   getShaders() {
-    return super.getShaders({vs, fs, modules: [project32, picking]});
+    return super.getShaders({vs, fs, modules: [layerUniforms, project32, picking, bitmapUniforms]});
```

Comment:
`layerUniforms` should be added to default modules?

https://github.com/visgl/deck.gl/blob/a6e8bc3be8d62bd499813d89f226582fbb650747/modules/core/src/lib/layer-manager.ts#L109